spring-data-neo4j/src/main/java/org/springframework/data/neo4j/repository/query/derived/builder/PropertyComparisonBuilder.java

```diff
@@ -79,4 +83,34 @@ private ComparisonOperator convertToComparisonOperator(Part.Type type) {
 				throw new IllegalArgumentException("No ComparisonOperator for Part.Type " + type);
 		}
 	}
+
+	/**
+	 * Sets the filter to ignore the case in case the underlying {@link Part} requires ignoring case and the property
+	 * actually supports it. Note: The method is modelled after {@link #setNestedAttributes(Part, Filter)} to have some
+	 * consistency. Preferable it should return a new filter.
+	 *
+	 * @param part
+	 * @param filter
+	 */
+	private void applyCaseInsensitivityIfShouldIgnoreCase(Part part, Filter filter) {
```

Comment:
We make pretty loose use of blank lines to group different parts of a method. Every method that has more than a single line of implementation uses a single blank line after the signature.